app/assets/stylesheets/home.scss

```diff
@@ -0,0 +1,49 @@
+@import "foundation_and_overrides";
+
+body {
+  background: url("Home_photo.jpg") no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
+
+.footer {
+  background: transparent;
+  text-align: center;
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  height: 4rem;
+  padding: 0.5rem;
+}
+  .footer .links a {
```

Comment:
Line should be indented 0 spaces, but was indented 2 spaces<br>Merge rule `.footer .links a` with rule on line 11<br>Selector should have depth of applicability no greater than 2, but was 3